### DISCUSSION TITLE
基于作者的win12 我基于Rust 编译成了exe安装版本，大家可以体验一下！

### DISCUSSION THREAD

**Discussion by pronouncing** [win12-online/win12#298] [Show and tell]
基于作者的WIN12我编译成了exe安装版，大家可以体验一下！
![image](https://github.com/tjy-gitnub/win12/assets/3447053/260f35c0-d95c-42e9-b15c-fdf41c60831d)

![image](https://github.com/tjy-gitnub/win12/assets/3447053/8e6b01ac-c342-421e-8fb0-e240571915c9)

WIN12安装版附件下载：
[windows12.zip](https://github.com/tjy-gitnub/win12/files/12545438/windows12.zip)

**Comment by NebulaTechs** [comment]
感谢！

**Comment by lingr7** [comment]
这个是怎么打包的？

**Reply by stxttkx** [reply]
他说是rust。。。我不知道。。。

**Comment by Britz-rabbit** [comment]
> 

应该是tauri吧

**Comment by ghost** [comment]
这么牛？！

**Comment by NB-Group** [comment]
再说亿遍：我们不支持任何有关windows本地win12应用，因为有PWA

**Comment by JasonMo1** [comment]
官方的发布是基于electron的吗？

**Reply by NB-Group** [reply]
好像是的，具体要问 @User782Tec
